expirationd/init.lua

```diff
@@ -682,6 +682,23 @@ local function default_iterate_with(task)
     end
 end
 
+-- ========================================================================= --
+-- Expiration daemon constants
+-- ========================================================================= --
+--
+-- {{{ Module constants
+--
+
+--- Module constants
+--
+-- @section Constants
+--
+
+--- Current module version in format `MAJOR.MINOR.PATCH`.
+_VERSION = require('expirationd.version')
```

Comment:
```suggestion
local _VERSION = require('expirationd.version')
```

Reply:
Hmm, If make this variable `local`, then `ldoc` will not document it. Now I have moved this constant to the `Module functions` section, which I have renamed to `Module functions and constants`

Reply:
Try:

```Lua
local _VERSION

---  doc comment
_VERSION
```

See:

https://github.com/tarantool/etcd-client/blob/master/etcd-client/init.lua#L10

Reply:
Yes, it worked, I returned to the previous doc version with a separate section for module constants